jobs/config.json

```diff
@@ -5275,6 +5275,24 @@
       "sig-cluster-lifecycle"
     ]
   },
+  "ci-kubernetes-e2e-kubeadm-gce-1-6-on-1-7": {
+    "args": [
+      "--cluster=",
+      "--env-file=jobs/platform/gce.env",
+      "--env-file=jobs/ci-kubernetes-e2e-kubeadm-gce-1-6-on-1-7.env",
+      "--kubeadm=ci",
+      "--kubernetes-anywhere-kubernetes-version=latest-1.6",
+      "--deployment=kubernetes-anywhere",
+      "--mode=local",
+      "--timeout=300m",
+      "--extract=ci/latest-1.6",
```

Comment:
Nope, this is intentional. The intent of the test is:

- Only run on the `release-1.7` branch
- Use the latest CI build of kubeadm (from that branch)
- Initialize a cluster using `kubeadm init --kubernetes-version latest-1.6`
- Run the ci/latest-1.6 Conformance tests against the cluster

This is because kubeadm allows you to initialize a cluster using several different versions of Kubernetes. The 1.7 version of kubeadm will install 1.7 by default, but supports installing back to 1.6 (when it became Beta).

Another way of looking at this is that kubeadm 1.6 fully supported installing Kubernetes 1.6 clusters. When we started adding kubeadm features for 1.7, we made sure that kubeadm could install Kubernetes 1.7, but wanted to maintain the backward compatibility of being able to install Kubernetes 1.6 as well. This was always supported, but we didn't have CI tests exercising the functionality.